README.md

```diff
@@ -1,8 +1,8 @@
-# Skyscanner Full-Stack Recruitment test
+# Skyscanner full-stack recruitment test
```

Comment:
I'm pretty sure that this sentence:


The zip file should be named {yourname}.zip, and should itself contain the frontend-recruitment-test project folder with your submission.

should read


The zip file should be named {yourname}.zip, and should itself contain the *full-stack*-recruitment-test project folder with your submission.